project/VS2010Express/XBMC.vcxproj

```diff
@@ -372,6 +372,7 @@
     <ClCompile Include="..\..\xbmc\filesystem\MultiPathFile.cpp" />
     <ClCompile Include="..\..\xbmc\filesystem\MusicDatabaseDirectory.cpp" />
     <ClCompile Include="..\..\xbmc\filesystem\MusicDatabaseDirectory\DirectoryNodeGrouped.cpp" />
+    <ClCompile Include="..\..\xbmc\filesystem\MusicDatabaseDirectory\MusicFileItemListModifier.cpp" />
```

Comment:
They are responsible for translating the `musicdb://` paths into the proper calls to `CMusicDatabase`.They are the implementation for the `musicdb://` VFS paths so IMO that's the right spot for them.